add line and polygon, point Phường 4

wsoft416 opened the pull request.

wsoft416 committed:
add line and polygon, point Phường 4

hiimBugCreator commented on index.html:
```diff
@@ -343,7 +369,7 @@
 				container: "viewDiv",
 				map: map,
-				center: [106.8033387, 10.8739831],
+				center: [106.68216950311093,10.772966281258045],
```
This line is not in your scope. Please rollback it. Thanks.

wsoft416 replied:
Done

hiimBugCreator commented on index.html:
```diff
@@ -206,28 +225,34 @@
+						[106.68682577057525, 10.775030447952853], 
+						[106.68481506507447, 10.772919119592833], 
+						[106.68209335660033, 10.770074710273892]
+					],
```
I think this list of coordinates is not really complete. Please zoom in on Google Maps to the maximum and get all the coordinates of the zigzag lines within your ability.

wsoft416 replied:
i had zoom and adjusted Coordinates

hiimBugCreator commented on index.html:
```diff
@@ -153,31 +170,33 @@
+					},
+					Name: "Quận 3",
+					Location: "Quận Thủ Đức",
+					popupTemplate: point_template_area
```
Move it to another file. Remove your comment. And what mean of Name: Quận 3 while Location: Quận Thủ Đức?

wsoft416 replied:
removed my comment  and changed Location to Quận 3

hiimBugCreator commented on index.html:
```diff
@@ -61,22 +78,22 @@
+							width: 1
+						}
+					},
+					popupTemplate: point_template_P4_hospital
```
rename point_template_P4_hospital to something like ward_4_poppup

wsoft416 replied:
i renamed to point_template_ward_4_poppup

hiimBugCreator commented on index.html:
```diff
@@ -61,22 +78,22 @@
+					longitude: 106.68127715688188, 
+					latitude: 10.774075083216818,
+					Name: "Bệnh viện Bình Dân",
+					Location: "Địa chỉ: 371 Điện Biên Phủ, Phường 4, Quận 3, Hồ Chí Minh 700000, Việt Nam",
```
You can show the text "Địa chỉ" in your popup like HTML. Please remove it.

wsoft416 replied:
So, what is content in location ?

hauletrung replied:
chỗ này em chỉ cần xoá cái chữ `Địa chỉ:` nha @wsoft416

wsoft416 replied:
Done

hiimBugCreator commented on index.html:
```diff
@@ -19,26 +19,43 @@
+						title: "{Name}",
+						content: "<b>{Location}</b>."
+					};
+
```
I think `point_template_area` is not in your scope. If i was wrong please ignore this comment.

wsoft416 replied:
point_template_area -> i didn't edit it. it still the same

hauletrung replied:
<img width="696" alt="Screenshot 2024-11-22 at 11 13 19 AM" src="https://github.com/user-attachments/assets/72ba7851-e8d2-4aa9-a964-81e205286edd">
chỗ này em nhìn vào cái file của em, chỗ đoạn xanh em bấm vô, nó sẽ xổ ra cái menu, em chọn Revert Change là được @wsoft416

wsoft416 replied:
Nhung đã lấy lại file và copy điều chỉnh lại như cũ chỉ khác là var đổi thành let

hiimBugCreator commented on index.html:
```diff
@@ -19,26 +19,43 @@
+		// var point_template_university = {
+		// 	title: "{Name}",
+		// 	content: "Trường ở <b>{Location}</b>."
+		// };
```
You can remove the commented code, or you can ignore it. So please remove the comment or remove the code.

wsoft416 replied:
I removed

hiimBugCreator reviewed (changes requested):
I have some comments for you. Please resolve it then notice me for the next review.

wsoft416 committed:
Change content according to requirement from pull request and zalo group

1. var -> let
2. remove comment declare variable
3. rollback long,lat map view
4. change variable ward popup
5. Change name location to Quận 3
6. adjust coordinate

hauletrung commented:
xong rồi phải resolve conflict nữa nha, em biết resolve conflict chưa @wsoft416 ?

hauletrung commented:
<img width="358" alt="Screenshot 2024-11-22 at 11 15 40 AM" src="https://github.com/user-attachments/assets/4e8d4eb5-a5ab-4cf0-8798-81badb8c8cf2">
để yêu cầu review lại, em nhìn ở góc phải bên trên như hình, bấm vào cái nút re-request review là được. Sau này tạo PR mới thì cũng add reviewer ở đây @wsoft416

wsoft416 committed:
Phục hồi lại content của point_template_area

wsoft416 committed:
Xử lý conflict
